### ISSUE TITLE
Clearance issue with the BSP adapter and finalized V1.5 gantry Z actuator

### ISSUE THREAD

**Issue by Fulg** [mzbotreprap/VORON#85]
I am pretty sure the BSP adapter was designed before the final Z actuator rocker, because there is no space for the adjustment screw (added with the last revision I believe).

In order to use it I had to trim the rocker a bit next to the adjustment screw, remove some plastic around the edges, and remove the raised middle Chimera screw mount so the screw does not interfere with the rocker movement.

I do not have STLs to share, this was done in-situ with a pair of side cutters :)

![img_1587](https://cloud.githubusercontent.com/assets/3320405/23440198/a04d93fa-fde8-11e6-8522-0ca493d02308.JPG)
![img_1588](https://cloud.githubusercontent.com/assets/3320405/23440200/a26eef12-fde8-11e6-9006-9b7e4d19cc76.JPG)

**Comment by mzbotreprap** [comment]
Well, it was designed with the final gantry, but when I was making the final tweaks, the whole assembly was off. This is the result. Just checked interference on the model, and I see the same thing.

I'll tweak the adapter vs. messing with the actuator arm. Don't really want to mess with that part.

**Comment by Fulg** [comment]
On my carriage this is worse because I used socket head screws. If I had used a button head screw for the middle Chimera screw (where I clipped the adapter) it would have also worked. 

All of my 20mm screws are socket head however and I didn't want to mix&match screw sizes for the hotend :)

**Comment by Fulg** [comment]
Are you sure this was fixed correctly? In the latest version of the V1.5 carriage assembly, the new BSP adapter still does not allow room for the middle Chimera screw, next to the rocker. I modified mine to get clearance for the third screw, before printing it this time, it ended up looking practically identical to the one I manually mutilated and photographed above.

![image](https://user-images.githubusercontent.com/3320405/28500262-fc6225f4-6f92-11e7-9c97-bb7d75b44eb3.png)
